aws-sns-lambda/module_test.go

```diff
@@ -0,0 +1,12 @@
+package test
```

Comment:
This logic was adapted from https://github.com/chanzuckerberg/cztack/tree/main/bless-ca/module_test.go